packages/css/src/breadcrumb/breadcrumb.scss

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+@mixin reset {
+  box-sizing: border-box;
+  break-after: avoid;
+  break-inside: avoid;
+  margin-block: 0;
+  padding-inline: 0;
+}
+
+.amsterdam-breadcrumb {
+  color: var(--amsterdam-breadcrumb-color);
+  font-family: var(--amsterdam-breadcrumb-font-family, inherit);
+  font-size: var(--amsterdam-breadcrumb-font-size-narrow);
+  line-height: var(--amsterdam-breadcrumb-line-height);
+  text-transform: capitalize;
+
+  @media screen and (width > $amsterdam-breakpoint) {
+    font-size: var(--amsterdam-breadcrumb-font-size-wide);
+  }
+}
+
+.amsterdam-breadcrumb__list {
+  column-gap: 0.5rem;
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+
+  @include reset;
+}
+
+.amsterdam-breadcrumb-item {
+  align-items: center;
+  display: flex;
+}
+
+.amsterdam-breadcrumb-item:not(:last-child)::after {
+  background-color: var(--amsterdam-breadcrumb-color);
+  content: "";
+  display: inline-block;
+  height: 16px;
+  margin-inline-start: 1rem;
+  -webkit-mask-image: url("../../../../proprietary/icons/static/ChevronRight.svg");
+  mask-image: url("../../../../proprietary/icons/static/ChevronRight.svg");
+  -webkit-mask-size: 16px;
+  mask-size: 16px;
+  width: 16px;
+}
+
+.amsterdam-breadcrumb-item__link {
+  color: inherit;
+  cursor: pointer;
+  outline-offset: var(--amsterdam-breadcrumb-item-link-outline-offset);
+  text-decoration: none;
```

Comment:
Shouldn't we give these links some padding? Currently, the hit area on small screens is ~25px, which is smaller than the [44px required by the WCAG](https://www.w3.org/TR/WCAG21/#target-size).

The `inList` Link variant has the same problem, maybe we should discuss this with design as well.

Reply:
done